src/calculateRentalCost.js

```diff
@@ -29,6 +29,20 @@
 
 function calculateRentalCost(days) {
   // write code here
+  const priceOfOneDay = 40;
+  const priceOfAllDays = priceOfOneDay * days;
```

Comment:
You better keep it simple
```suggestion
  const dayCost = 40;
  const totalCost = dayCost * days;
```